resources/views/maps/groups/edit.blade.php

```diff
@@ -27,20 +27,47 @@
         @endif
         <div class="panel-body">
```

Comment:
Can't use "panel-body" when in a modal, it creates a weird border. Need to use "modal-body"
![image](https://user-images.githubusercontent.com/3263440/196011072-87b9c830-e0bf-49c3-a660-9ed1409dcaaf.png)